src/main/java/io/github/bobisawesome07/world/dimension/ModDimensions.kt

```
@@ -0,0 +1,100 @@
+     * Registers the pocket dimension type during game bootstrap
+     *
+     * @param context Registry context
+     */
+    fun bootstrapType(context: Registerable<DimensionType?>?) {
+        register()
+    }
+}
+
+private fun register() {
+
+}
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)